src/api/python/cvc5.pxi

```diff
@@ -686,28 +691,33 @@ cdef class Solver:
                 Op mkOp(Kind kind, uint32_t arg0, uint32_t arg1)
         '''
         cdef Op op = Op(self)
+        cdef vector[int] v
 
-        if arg0 is None:
+        if len(args) == 0:
             op.cop = self.csolver.mkOp(k.k)
-        elif arg1 is None:
-            if isinstance(arg0, kind):
-                op.cop = self.csolver.mkOp(k.k, (<kind?> arg0).k)
-            elif isinstance(arg0, str):
+        elif len(args) == 1:
+            if isinstance(args[0], kind):
```

Comment:
I don't see a corresponding function in the cpp API. If the second argument exists, it is either a string, an int, or a vector, no?

Reply:
The old function have that case, I don't know why. Anyway I deleted it now.